config/cscs.py

```diff
@@ -1073,6 +1073,22 @@
                 '--tag=production',
                 '--timestamp=%F_%H-%M-%S'
             ]
+        },
+        {
+            'name': 'production',
+            'options': [
+                '--unload-module=reframe',
+                '--exec-policy=async',
+                '--strict',
+                '--output=$SCRATCH/$USER/regression/production',
+                '--perflogdir=$SCRATCH/$USER/regression/production/logs',
+                '--stage=$SCRATCH/regression/production/stage',
```

Comment:
Since everywhere the prefix is the same here, perhaps it would make sense to simply set the `--prefix=$SCRATCH/$USER/regression/production`